pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [build-system]
 build-backend = "setuptools.build_meta"
 requires = [
-    "cython>=0.29",
-    "numpy==2.0.0", #oldest-supported-numpy is 1.x and hollerith supports only 2.0+
+    "cython==3.0.11",
+    "numpy==2.1.3",
```

Comment:
Let's not pin down versions for libraries... we should make them more flexible. Otherwise hollerith will force all of us to use this specific cython and numpy version when installed together with other libraries

Reply:
Ah wait this is for build only